solved lab_1

Lion2555 opened the pull request.

Lion2555 committed:
solved lab_1

Lion2555 committed:
fixed

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,213 @@
+            cout << "Enter flight: ";
+            cin >> baggages[i].flight;
+            cout << "you have successfully modify information about your Baggage. \n";
+            return;
```
do you need this return?

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,213 @@
+    for (int i = 0; i < numBaggages; ++i)
+    {
+        cout << endl;
+        cout << "Owner Name: " << baggages[i].ownerName << endl;
+        cout << "Weight: " << baggages[i].weight << "kg" << endl;
+        cout << "Number of Seats: " << baggages[i].numberOfSeats << endl;
+        cout << "Flight: " << baggages[i].flight << endl;
```
you can skip cout keyword for each new line

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,213 @@
+void SaveBaggagesToFile()
+{
+    ofstream outFile("baggages.bin", ios::binary | ios::out);
+    if (!outFile)
+    {
+        cout << "Unable to open file.\n";
+        return;
+    }
+    outFile.write((char*)&numBaggages, sizeof(numBaggages));
+    outFile.write((char*)baggages, numBaggages * sizeof(Baggage));
+    outFile.close();
+    cout << "Baggages saved to file successfully.\n";
```
if/else

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,213 @@
+void LoadBaggagesFromFile()
+{
+    ifstream inFile("baggages.bin", ios::binary | ios::in);
+    if (!inFile)
+    {
+        cout << "Unable to open file or file does not exist.\n";
+        return;
+    }
+    inFile.read((char*)&numBaggages, sizeof(numBaggages));
+    inFile.read((char*)baggages, numBaggages * sizeof(Baggage));
+    inFile.close();
+
+    cout << "Baggages loaded from file successfully.\n";
```
if/else

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,213 @@
+        << "Enter your choice: ";
+}
+
+int main()
```
too 

many

blank

lines

vloooo reviewed (commented):
fix the changes and re-request the review

Lion2555 committed:
fixed#2

vloooo commented on main.cpp:
```diff
@@ -0,0 +1,206 @@
+#include <iostream>
```
let's discuss the PR in the class

Lion2555 committed:
feat(core): add centralized config and bootstrap helpers

- config.php: constants (charset, upload dir, size limit, allowed EXT/MIME, PHP error map)
- bootstrap.php: e() escaper, ensure_upload_dir()
